xml/System.Diagnostics/Activity.xml

```diff
@@ -172,8 +172,8 @@
         <ReturnType>System.TimeSpan</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>If the <see cref="T:System.Diagnostics.Activity" /> has ended (<see cref="M:System.Diagnostics.Activity.Stop" /> or <see cref="M:System.Diagnostics.Activity.SetEndTime(System.DateTime)" /> was called) then this is the delta between <see cref="P:System.Diagnostics.Activity.StartTimeUtc" /> and end. If the <see cref="T:System.Diagnostics.Activity" /> is not ended and <see cref="M:System.Diagnostics.Activity.SetEndTime(System.DateTime)" /> was not called then this is <see cref="F:System.TimeSpan.Zero" />.</summary>
```

Comment:
You should replace `If the activity is not ended` with `If the activity has not ended`

Reply:
```suggestion
        <summary>If the <see cref="T:System.Diagnostics.Activity" /> has ended (<see cref="M:System.Diagnostics.Activity.Stop" /> or <see cref="M:System.Diagnostics.Activity.SetEndTime(System.DateTime)" /> was called) then this is the delta between <see cref="P:System.Diagnostics.Activity.StartTimeUtc" /> and end. If the <see cref="T:System.Diagnostics.Activity" /> has not ended and <see cref="M:System.Diagnostics.Activity.SetEndTime(System.DateTime)" /> was not called then this is <see cref="F:System.TimeSpan.Zero" />.</summary>
```